src/lib/pubkey/kyber/kyber_common/kyber.cpp

```diff
@@ -71,6 +71,20 @@ KyberMode::Mode kyber_mode_from_string(std::string_view str) {
    throw Invalid_Argument(fmt("'{}' is not a valid Kyber mode name", str));
 }
 
+/**
+ * Constant time implementation for computing an unsigned integer division
+ * with KyberConstants::Q = 3329.
+ *
+ * It enforces the optimization of various compilers,
+ * replacing the division operation with multiplication and shifts.
+ *
+ * @returns (a / KyberConstants::Q)
+ */
+uint16_t int_div_kyber_q(uint32_t a) {
```

Comment:
Nit: functions that carry some constant-time guarantee are prefixed with `ct_` by convention. Maybe so:

```suggestion
uint16_t ct_int_div_kyber_q(uint32_t a) {
```